File: LeetCode/Arrays/66. Plus One.py
class Solution:
    def plusOne(self, digits: List[int]) -> List[int]:
        st = ""
        for i in digits:
            st+=str(i)
        num = int(st)+1
        output = []
        for j in str(num):
            output.append(int(j))
        return output

File: LeetCode/Arrays/66. Plus One.swift
class Solution {
    func plusOne(_ digits : [Int]) -> [Int] {
        var arr = digits
        for i in (0..<arr.count).reversed() {
            if (arr[i]<9) {
                arr[i] += 1
                return arr
            } else {
                arr[i] = 0
            }
        }
        arr.insert(1, at: 0)
        return arr
    }
}
